docs/how-to/kafka-ui.md

```diff
@@ -0,0 +1,144 @@
+
+- View Apache Kafka cluster configuration, topics, ACLs, consumer groups and more
+- Broker performance monitoring via JMX metrics dashboards
+- Seamless integration with other Charmed Apache Kafka operators, like [Charmed Apache Kafka Connect](https://charmhub.io/kafka-connect) and [Charmed Karapace](https://charmhub.io/karapace)
+
+In this guide, you will deploy the Charmed Kafka UI operator, integrate it with the Charmed Apache Kafka and other related products, and enable authentication and TLS for securing access to the Kafka UI.
+
+## Prerequisites
+
+It is assumed that you have a running Apache Kafka cluster using the Charmed Apache Kafka operator. If you are unsure on how to achieve that, it is recommended to review the [Deploy Apache Kafka](tutorial-deploy) tutorial first. The output of the `juju status` command for a cluster running 3 brokers and 3 KRaft controllers will be:
+
+```text
+Model  Controller  Cloud/Region         Version  SLA          Timestamp
+ui     lxd         localhost/localhost  3.6.9    unsupported  08:37:59+01:00
+
+App         Version  Status  Scale  Charm  Channel  Rev  Exposed  Message
+controller  4.0.0    active      3  kafka  4/edge   239  no
+kafka       4.0.0    active      3  kafka  4/edge   239  no
+
+Unit           Workload  Agent  Machine  Public address  Ports      Message
+controller/0   active    idle   3        10.160.219.43   9098/tcp
+controller/1*  active    idle   4        10.160.219.30   9098/tcp
+controller/2   active    idle   5        10.160.219.64   9098/tcp
+kafka/0        active    idle   0        10.160.219.42   19093/tcp
+kafka/1        active    idle   1        10.160.219.90   19093/tcp
+kafka/2*       active    idle   2        10.160.219.114  19093/tcp
+
+Machine  State    Address         Inst id        Base          AZ  Message
+0        started  10.160.219.42   juju-a00eba-0  ubuntu@24.04      Running
+1        started  10.160.219.90   juju-a00eba-1  ubuntu@24.04      Running
+2        started  10.160.219.114  juju-a00eba-2  ubuntu@24.04      Running
+3        started  10.160.219.43   juju-a00eba-3  ubuntu@24.04      Running
+4        started  10.160.219.30   juju-a00eba-4  ubuntu@24.04      Running
+5        started  10.160.219.64   juju-a00eba-5  ubuntu@24.04      Running
+```
+
+## Deploy charmed Kafka UI
+
+To deploy the charmed Kafka UI operator, simply use the following command:
+
+```bash
+juju deploy kafka-ui --channel latest/edge
+```
+
+Once the charmed Kafka UI operator is deployed, it will end up in `blocked` state, since it needs to be integrated with a charmed Apache Kafka cluster. The output of `juju status` command will be like below:
+
+```text
+...
+kafka-ui/0*    blocked   idle   6        10.160.219.25              application needs Kafka client relation
+...
+```
+
+## Integrate charmed Kafka UI with Apache Kafka
+
+To activate the charmed Kafka UI application, integrate it with the charmed Apache Kafka application:
```

Comment:
Suggestion: If charmed is used as part of a proper name, then it must be capitalised: Charmed Kafka UI application.

```suggestion
To activate the Charmed Kafka UI application, integrate it with the Charmed Apache Kafka application:
```